pre_commit_hooks/file_contents_sorter.py

```diff
@@ -23,12 +23,19 @@
 def sort_file_contents(
     f: IO[bytes],
     key: Optional[Callable[[bytes], Any]],
+    unique: bool,
```

Comment:
this should be keyword only, otherwise it's a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap)

Reply:
this didn't get fixed, here's a vid where I talk about [all the parameter types](https://youtu.be/aKCfCmSggPg)